website/docusaurus.config.js

```diff
@@ -146,23 +146,17 @@ const config = {
               {
```

Comment:
Isn't it possible to add a custom class to the entire column in the config, @IrbisKronos ? I'm worried about from where you take `.socialNetLinks`.